it/v4/endpoints/bfLoss/amend/AmendBFLossControllerISpec.scala

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v4.endpoints.bfLoss.amend
+
+import api.hateoas.HateoasLinks
+import api.models.errors._
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status
+import play.api.http.Status._
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import support.V4IntegrationBaseSpec
+import support.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+
+class AmendBFLossControllerISpec extends V4IntegrationBaseSpec {
+
+  val lossAmount = 2345.67
+
+  val downstreamResponseJson: JsValue = Json.parse(s"""
+                                                      |{
```

Comment:
I guess these are just being reinstated, but while you're there, can you move this block over to the left so it's in line with the requestJson one below it?

+ any others that are like that

Reply:
updated!